kpi/mixins/xls_exportable.py

```diff
@@ -35,7 +35,7 @@ def ordered_xlsform_content(self,
         )
         return content
 
-    def to_xls_io(self, versioned=False, **kwargs):
+    def to_xlsx_io(self, versioned=False, **kwargs):
```

Comment:
Maybe out of scope for this PR but what about renaming `XlsExportableMixin` to `XlsxExportableMixin'. 
Should we change it in this PR or open another issue?